.github/workflows/test.yml

```
@@ -65,3 +65,46 @@ jobs:
         uses: actions/checkout@v2
       - name: Check Go module consistency
         run: ./tools/check_go_mod.sh
+
+  build:
+    name: Build binary
```

```suggestion
    name: Build code
```
We're building multible non-executable binary object files here so name _binary_ doesn't sound right for me.

---

Done